tekton/src/pipelines/taskdefs/fvt-manage/phase1.yml.j2

```diff
@@ -32,3 +28,25 @@
       value: monitoring
   runAfter:
     - fvt-manage-usersync
+# Manage FVT Ancestors
+- name: fvt-manage-ancestors
+  {{ lookup('template', 'taskdefs/fvt-manage/api/taskref.yml.j2') | indent(2) }}
+  params:
+    {{ lookup('template', 'taskdefs/fvt-manage/api/params.yml.j2') | indent(4) }}
+    - name: fvt_test_suite_prefix
+      value: core
+    - name: fvt_test_suite
+      value: ancestors
+  runAfter:
+    - fvt-manage-bvt
```

Comment:
the name of setup task is fvt-manage-setup, even though suite name has changed to be bvt instead. Here we need to stay with that name: https://github.com/ibm-mas/cli/pull/739/files#diff-c0bd12dab72895b7b1363d8040a9c07bad8212f5f54d52105292a88d4756b8ffR10

Also, if ancestors and automationscripts depend on setup they cannot be in phase 1 and should be moved to a later phase (please move to the ones with less tests).

Once they are moved, the runAfter list must also be updated to follow other tests in the same phase as well as the tests in the next phase must consider the two new tests in their own "runAfter" list